spark/src/test/java/org/apache/iceberg/spark/source/TestSparkTableUtil.java

```diff
@@ -116,4 +123,28 @@ public void testPartitionScanByFilter() {
     Dataset<Row> partitionDF = SparkTableUtil.partitionDFByFilter(spark, qualifiedTableName, "data = 'a'");
     Assert.assertEquals("There should be 1 matching partition", 1, partitionDF.count());
   }
+
+  @Test
+  public void testImportPartitionedTable() throws Exception {
```

Comment:
Currently, it creates a hadoop table inside. I can change the signature to accept `Table` parameter so that we can import as the specific table.  you mean test import as a hadoop table and import as a hive table, right?